include/envoy/upstream/cluster_manager.h

```diff
@@ -75,6 +75,7 @@ class ClusterManager {
    */
   virtual Http::ConnectionPool::Instance* httpConnPoolForCluster(const std::string& cluster,
                                                                  ResourcePriority priority,
+                                                                 enum Http::Protocol protocol,
```

Comment:
nit: C++ does not require `enum`. Please remove. Same elsewhere.

Reply:
Fixed.